### PR TITLE
Feature/Song notification

### PR DESCRIPTION
**Pull request by Yavor-Ivanov**
I added a desktop notification when the song changes. This is what it looks like:

screen shot 2016-11-21 at 4 11 56 pm

I use the video thumbnail address from issue #20.

### CODE REVIEW

File: public/client.html [hunk 1]
```diff
@@ -36,6 +36,6 @@
 </div>
 
 <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+<script src="https://www.youtube.com/iframe_api"></script>
```
Comment (Bratanov) [inline public/client.html:39]: The order of the imports here was intended. We had cases in which the callback defined here:  https://github.com/Yavor-Ivanov/community-driven-radio/blob/4f0c6bfcd393d6a9cacfc68bde1868e2e0d02d6e/public/badass-client.js#L164 for the YouTube iframe API, was sometimes not called when  `https://www.youtube.com/iframe_api` loads before our JS files.

Did you have any experiences with this, or it was moved for aesthetics?

**Comment by Yavor-Ivanov** [comment]
I think something went wrong during the merge and it got reverted. I don't remember changing the order.

File: public/badass-client.js [hunk 2]
```diff
@@ -43,12 +43,33 @@ socket.on('chat_msg', function(data){
 });
 
 socket.on('new_song', function(song) {
-	$('#text').append('<p>Now plaing: ' + song + '</p>');
+	var recv = song;
+	console.log(recv);
```
Comment (IvayloAtanasov) [inline public/badass-client.js:47]: To be removed

File: public/badass-client.js [hunk 3]
```diff
@@ -43,12 +43,33 @@ socket.on('chat_msg', function(data){
 });
 
 socket.on('new_song', function(song) {
-	$('#text').append('<p>Now plaing: ' + song + '</p>');
+	var recv = song;
+	console.log(recv);
+	var song = queryStringToObj(song.url_params);
+
+	$('#text').append('<p>Now plaing: ' + recv.info.title + '</p>');
```
Comment (IvayloAtanasov) [inline public/badass-client.js:50]: Typo. This goes waaay back in history.

File: public/badass-client.js [hunk 4]
```diff
@@ -43,12 +43,33 @@ socket.on('chat_msg', function(data){
...
+	console.log(recv);
+	var song = queryStringToObj(song.url_params);
+
+	$('#text').append('<p>Now plaing: ' + recv.info.title + '</p>');
 	$('#text').scrollTop(999999999);
 
-	// NOTE <Yavor>: Maybe just send JSON from the server?
-	var song = queryStringToObj(song);
 	player.play(song.url, song);
+	send_notification = function(song) {
```
Comment (IvayloAtanasov) [inline public/badass-client.js:54]: to be renamed to sendNotification and made a local var

**Pull request merged**